modules/gateways/xendit.php

```diff
@@ -10,12 +10,12 @@
 require __DIR__ . '/xendit/autoload.php';
 
 // defines
```

Comment:
since we are not using `define` maybe can remove this line?

Reply:
Updated @ikanyu